src/raytracing/raytracing.c

```
@@ -72,25 +72,25 @@ bool is_in_shadow(t_vec hit_point, t_light *light, t_scene *scene)
 	t_vec		ray_origin;
-	t_object	*closest_object;
+	t_object	*closest_obj;
 	t_vec		hit_point;
 	double		t_closest;
 
 	ray_origin = mlx->scene->camera.position;
 	ray_dir = generate_ray_dir(&mlx->scene->camera, x, y, &mlx->img);
 	closest_object = find_closest_object(mlx->scene,
 			ray_origin, ray_dir, &t_closest);
-	if (closest_object)
+	if (closest_obj)
```

Cppcheck (2.10-2)

[error] Uninitialized variable

(uninitvar)